mod5hw/макет Cadastre/css/style.css

```diff
@@ -0,0 +1,232 @@
+@font-face {
+    font-family: 'Helvetica Neue Regular';
+    src: url('../fonts/helveticaneuecyr-thin-webfont.eot');
+    src: url('../fonts/helveticaneuecyr-thin-webfont.eot?#iefix') format('embedded-opentype'),
+         url('../fonts/helveticaneuecyr-thin-webfont.woff2') format('woff2'),
+         url('../fonts/helveticaneuecyr-thin-webfont.woff') format('woff'),
+         url('../fonts/helveticaneuecyr-thin-webfont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Helvetica Neue Regular';
+    src: local('Helvetica Neue Bold'), local('Helvetica-Neue-Bold'),
+        url('../fonts/HelveticaNeueCyr-Bold.woff2') format('woff2'),
+        url('../fonts/HelveticaNeueCyr-Bold.woff') format('woff'),
+        url('../fonts/HelveticaNeueCyr-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+  }
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: "Helvetica Neue Regular", Helvetica, sans-serif;
+}
+
+.navigation-panel__horisontal-list {
+    padding: 0;
+    display: flex;
+    justify-content: flex-end;
+    list-style: none;
+    background-color: #085393;
+}
+
+.navigation-panel__horisontal-list li {
+    padding: 10px 15px;
+    color: #54a8e1;
+}
+
+.navigation-panel__horisontal-list a {
+    text-decoration: none;
+    color: #54a8e1;
+}
+
+.navigation-panel__horisontal-list li:hover {
+    background-color: #4eabda;
+    color: white;
+}
+
+.navigation-panel__horisontal-list a:hover {
+    color: white;
+}
+
+.navigation-panel__horisontal-list a:visited {
+    color: white;
+}
+
+.fas.fa-flag {
+    margin-right: 10px;
+}
+
+.offers__li {
+    border-left: 1px solid #56a2dc;
+    color: white;
+}
+
+.hello__li.hello__li {
+    margin-right: 20px;
+    color: white
+}
+
+.offers__li:hover, .hello__li:hover {
+    background: linear-gradient(0deg, rgba(46,122,196,1) 0%, rgba(81,177,220,1) 100%);
+}
+
+.enter__a, .register__a {
+    text-decoration: none;
+    color: white;
+}
+
+.enter__a:visited, .register__a:visited {
+    color: white;
+}
+
+.enter__a:hover, .register__a:hover {
+    text-decoration: underline;
+}
+
+main {
+    background-image: url(../img/background.png);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    height: 1028px;
+    display: flex;
+    color: white;
+}
+
+.content-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    max-width: 550px;
+}
+
+.site-name {
+    padding: 15px 0;
+    margin-bottom: 2px;
+    text-align: center;
+    background-color: rgba(0, 0, 0, 0.5);
```

Comment:
I still don't understand why you can't add bg color to the parent and not add it to every child